parcel_test.go

```diff
@@ -31,57 +32,89 @@ func getTestParcel() Parcel {
 // TestAddGetDelete проверяет добавление, получение и удаление посылки
 func TestAddGetDelete(t *testing.T) {
 	// prepare
-	db, err := // настройте подключение к БД
+	db, err := sql.Open("sqlite", "tracker.db")
+	require.NoError(t, err)
+	defer db.Close()
+
 	store := NewParcelStore(db)
 	parcel := getTestParcel()
 
-	// add
-	// добавьте новую посылку в БД, убедитесь в отсутствии ошибки и наличии идентификатора
+	number, err := store.Add(parcel)
+	require.NoError(t, err)
+	assert.NotEmpty(t, number)
+
+	newParcel, err := store.Get(number)
+	require.NoError(t, err)
+	assert.Equal(t, parcel.Address, newParcel.Address)
+	assert.Equal(t, parcel.Client, newParcel.Client)
+	assert.Equal(t, parcel.CreatedAt, newParcel.CreatedAt)
+	assert.Equal(t, parcel.Status, newParcel.Status)
 
-	// get
-	// получите только что добавленную посылку, убедитесь в отсутствии ошибки
-	// проверьте, что значения всех полей в полученном объекте совпадают со значениями полей в переменной parcel
+	err = store.Delete(number)
+	require.NoError(t, err)
+
+	_, err = store.Get(number)
+	require.Equal(t, sql.ErrNoRows, err)
```

Comment:
Для сравнивания ошибок нужно использовать специализированную функцию
```suggestion
	require.ErrorIs(t, err, sql.ErrNoRows)
```